core/modules/block/block.module

```diff
@@ -266,6 +275,7 @@ function block_custom_block_save(array $edit, $delta = NULL) {
       'delta' => $delta,
       'info' => '',
       'title' => '',
+      'description' => '',
```

Comment:
...you also need to add this in https://github.com/backdrop/backdrop/blob/1.x/core/modules/block/block.module#L176 ...I believe that this might be what is causing the "Undefined index: description" error in the tests.

Reply:
Done. Thanks @klonos for your comments on this.